db/BL1611-ID-1-EnergyAddon.db

```diff
@@ -0,0 +1,318 @@
+# ENERGY
+#####################################
+record( ao, "AM1611-4-I10:energy:eV:fbk")
```

Comment:
Yeah, at least the ones we set up. The controls group do much more with them, but that is a basic double type epics record.